src/physics/base/PhysicsParams.cc

```diff
@@ -237,7 +237,12 @@ void PhysicsParams::build_ids(const ParticleParams& particles,
             data->hardwired.photoelectric              = process_id;
             data->hardwired.photoelectric_table_thresh = units::MevEnergy{0.2};
             data->hardwired.livermore_pe               = ModelId{model_idx};
-            data->hardwired.livermore_pe_params = pe_model->device_pointers();
+            // TODO: fix me
+#ifdef __CUDACC__
+            data->hardwired.livermore_pe_data = pe_model->device_pointers();
+#else
+            data->hardwired.livermore_pe_data = pe_model->host_pointers();
+#endif
```

Comment:
Oh yuck -- I didn't think about this. This will crash when we try to run on device because inside this `.cc`, `__CUDACC__` is always undefined, so the GPU version will get host pointers.

I think the easy fast solution is to change `detail::LivermorePEPointers` in the PhysicsInterface to `LivermorePEData<W, M>`. It'll cause a duplicate copy of the livermore data to be owned by the physics, but the livermore data isn't huge so I'm not too worried -- let's go for "correct" for the time being.

Reply:
Thanks, I was stuck on that :). Sounds good!